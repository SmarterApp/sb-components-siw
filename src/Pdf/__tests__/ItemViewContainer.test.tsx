import * as React from "react";
import * as ReactDOM from "react-dom";
import { ItemViewContainer } from "../ItemViewContainer";
import { shallow, mount, render } from "enzyme";
import { itemGroup } from "./PdfMocks";

describe("EvidenceStatement", () => {
  it("matches snapshot with score", () => {
    expect(
      shallow(
        <ItemViewContainer itemData={itemGroup} displayScoreInfo={true} />
      )
    ).toMatchSnapshot();
  });
  it("matches snapshot without score", () => {
    expect(
      shallow(
        <ItemViewContainer itemData={itemGroup} displayScoreInfo={false} />
      )
    ).toMatchSnapshot();
  });
});
