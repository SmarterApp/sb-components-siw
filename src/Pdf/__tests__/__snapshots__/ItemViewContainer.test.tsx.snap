// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EvidenceStatement matches snapshot with score 1`] = `
<div
  className="page"
>
  <QuestionView
    displayScoreInfo={true}
    key="1"
    question={
      Object {
        "data": Object {
          "associatedItems": "string",
          "commonCoreStandardsDescription": "string",
          "depthOfKnowledge": "string",
          "educationalDifficulty": "string",
          "evidenceStatement": "string",
          "itemCardViewModel": Object {
            "bankKey": 1,
            "claimCode": "234536",
            "claimLabel": "Math Claim",
            "commonCoreStandardId": "",
            "depthOfKnowledge": "",
            "domain": "MATH",
            "grade": 8,
            "gradeLabel": "grade 3",
            "interactionTypeCode": "",
            "interactionTypeLabel": "",
            "isPerformanceItem": false,
            "itemKey": 1,
            "subjectCode": "MATH",
            "subjectLabel": "math",
            "targetDescription": "Description",
            "targetHash": 212,
            "targetId": "23463467",
            "targetShortName": "1-3",
            "title": "",
          },
          "targetDescription": "string",
        },
        "id": "12334",
        "questionNumber": 1,
        "view": Object {
          "captured": true,
          "html": undefined,
          "id": "12334",
          "picturePath": undefined,
          "type": 1,
        },
      }
    }
  />
</div>
`;

exports[`EvidenceStatement matches snapshot without score 1`] = `
<div
  className="page"
>
  <QuestionView
    displayScoreInfo={false}
    key="1"
    question={
      Object {
        "data": Object {
          "associatedItems": "string",
          "commonCoreStandardsDescription": "string",
          "depthOfKnowledge": "string",
          "educationalDifficulty": "string",
          "evidenceStatement": "string",
          "itemCardViewModel": Object {
            "bankKey": 1,
            "claimCode": "234536",
            "claimLabel": "Math Claim",
            "commonCoreStandardId": "",
            "depthOfKnowledge": "",
            "domain": "MATH",
            "grade": 8,
            "gradeLabel": "grade 3",
            "interactionTypeCode": "",
            "interactionTypeLabel": "",
            "isPerformanceItem": false,
            "itemKey": 1,
            "subjectCode": "MATH",
            "subjectLabel": "math",
            "targetDescription": "Description",
            "targetHash": 212,
            "targetId": "23463467",
            "targetShortName": "1-3",
            "title": "",
          },
          "targetDescription": "string",
        },
        "id": "12334",
        "questionNumber": 1,
        "view": Object {
          "captured": true,
          "html": undefined,
          "id": "12334",
          "picturePath": undefined,
          "type": 1,
        },
      }
    }
  />
</div>
`;
